src/ApiCalendar.ts

```diff
@@ -82,19 +87,30 @@ class ApiCalendar {
 
   /**
    * Sign in Google user account
+   * @returns {any} Promise resolved if authentication is successful, rejected if unsuccessful.
    */
-  public handleAuthClick(): void {
+  public handleAuthClick(): any {
```

Comment:
If I'm not mistaken the right return type is `Promise<void>` no ?

Reply:
Correct, fixed.